src/pyqasm/visitor.py

```diff
@@ -952,7 +968,16 @@ def _visit_custom_gate_operation(
         # Update the depth only once for the entire custom gate
         if self._recording_ext_gate_depth:
             self._recording_ext_gate_depth = False
-            self._update_qubit_depth_for_gate([op_qubits], ctrls)
+            if not self._in_branching_statement:  # if custom gate is not in branching statement
+                self._update_qubit_depth_for_gate([op_qubits], ctrls)
+            else:
+                for ops in [op_qubits] + [ctrls]:  # get qubit registers in branching operations
```

Comment:
`op` and `ops` is a little bit confusing given the fact that we are referring to qubit registers here. Can we change this (and [here](https://github.com/qBraid/pyqasm/pull/200/files#diff-802ca57af29fc92ca8eb2f536fb5217c61c874c4c35bd39b7fd82512ee0e9a84R865)) to `qubit_subset` and `qubit` instead?

Reply:
How was this resolved?